admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/service/impl/TemplateServiceImpl.java

```diff
@@ -218,10 +217,10 @@ public TemplateResponseDto getAllTemplateByLanguageCodeAndTemplateTypeCode(Strin
 	public IdAndLanguageCodeID createTemplate(TemplateDto template) {
 
 		Template templateEntity;
-
+		String uniqueId = null;
 		try {
-			if (StringUtils.isNotEmpty(supportedLang) && supportedLang.contains(template.getLangCode())) {
-				String uniqueId = generateId();
+			if (template.getId().isBlank()) {
```

Comment:
if someone sends  "     "  is blank will be false trim it before

Reply:
taken care

Reply:
StringUtils.isBlank("  ")       = true